### PR TITLE
Typos, Markdown tweaks, and minor clarifications.

### PR DESCRIPTION
**Pull request by rnewman**
@linuxwolf, I fixed a few things as I read. The removed newlines should fix some markdown spacing niggles. Lemme know what you think. I'll add you as a collaborator in case you want to strip out some unwanted changes by pushing to this branch.

### CODE REVIEW

**Comment by rnewman** [comment]
Here's how this looks with my changes:

https://github.com/rnewman/mozilla-lockbox.github.io/blob/rnewman/doc-errata/architecture/sync.md

File: architecture/sync.md [hunk 1]
```diff
@@ -409,21 +382,18 @@ The following are performed prior to the rest of the sync process to prepare an
 
 1. A read/write IndexedDB transaction is opened against the `pending` and stable collcetions (`items` and `keystores`).
```
Comment (pdehaan) [inline architecture/sync.md]: typo: "collections"

File: architecture/sync.md [hunk 2]
```diff
@@ -409,21 +382,18 @@ The following are performed prior to the rest of the sync process to prepare an
...
         4. The `record` property is set to a clone of this stable record.
-
     2. The new `pending` record is inserted/updated.
 
 3. The IndexedDB transaction is committed.
 
 ## Occurrence and Frequency
 
-Lockbox automatically attempts to sync the user's data between the device and remote storage transparently and unobtrusively.  Additionally, Lockbox provides a action that lets the user to immediately trigger a sync.
+Lockbox automatically attempts to sync the user's data between the device and remote storage transparently and unobtrusively.  Additionally, Lockbox provides a action that lets the user immediately trigger a sync.
```
Comment (pdehaan) [inline architecture/sync.md]: "an action"?

File: architecture/sync.md [hunk 3]
```diff
@@ -467,7 +437,7 @@ The following telemetry event is used to record sync interactions.
 
 ## Schema Changes
 
-The on-device IndexedDB database originally documented in [data storage]()
+The on-device IndexedDB database originally documented in [data storage]().
```
Comment (pdehaan) [inline architecture/sync.md]: nit: missing URL

File: architecture/sync.md [hunk 4]
```diff
@@ -82,15 +82,15 @@ Changes are first placed in the `pending` collection before they are applied to
...
 
-When an item or keystore is changed locally, a record is inserted/updated into the `pending` collection rather than directly applying to the `items` or `keystores`.  When remote changes are fetched, a record for each is inserted/updated into the `pending` collection.
+When an item or keystore is changed locally, a record is inserted/updated into the `pending` collection rather than directly applying to the `items` or `keystores` collections.  When remote changes are fetched, a record for each is inserted/updated into the `pending` collection.
 
 ### Markers
 
-In addition to tracking the actual changes, the markers (i.e., [server timestamps](#server-timestamps)) are also tracked locally. The marker for each collection is stored on the device in IndexedDB via the `markers` collection:
+In addition to tracking the actual changes, the markers (_i.e._, [server timestamps](#server-timestamps)) are also tracked locally. The marker for each collection is stored on the device in IndexedDB via the `markers` collection:
```
Comment (pdehaan) [inline architecture/sync.md]: Should this be "i.e." or "e.g." (to match change above on L28)?

Comment (linuxwolf) [reply architecture/sync.md]: this really is a case of "that is" (i.e.) rather than "for instance" (e.g.).

File: architecture/sync.md [hunk 5]
```diff
@@ -190,15 +178,13 @@ Verification is performed as follows:
...
-
   - If this request fails with an HTTP error code of **403 (Unauthorized)**, an attempt is made to refresh it.
   - If this request succeeds, the user's authorization is valid.
 
 Expired, invalidated, or missing access token can be refreshed as follows:
 
 1. The cached refresh token is examined; if there is no refresh token, any existing access token is cleared from device cache and verification fails with a `AUTH` error.
-2. A POST HTTPS request is made to the FxA OAuth token endpoint, with a grant_type of "refresh_token" and including the cached refresh token:
-
+2. A `POST` HTTPS request is made to the FxA OAuth token endpoint, with a `grant_type` of "refresh_token" and including the cached refresh token:
```
Comment (pdehaan) [inline architecture/sync.md]: Below (L199), we say _"An HTTP "GET" request"_. Not sure if we want to change this to:

> _"An HTTPS "POST" request is made ..."_

or change L199 below to

> _"A `GET` HTTPS request is made ..."_

Comment (linuxwolf) [reply architecture/sync.md]: they should be consistent, but I don't have a preference with which way to be consistent.

**Comment by linuxwolf** [comment]
@rnewman Thanks again for the patch.  I have some further corrections based on @pdehaan's feedback on [my fork](https://github.com/linuxwolf/mozilla-lockbox.github.io/commit/1eb98d07cc13159d551cb598b87b4c77dfc3a70a).  Unfortunately, github doesn't want to let me push to your fork (-:

**Comment by rnewman** [comment]
No worries! Continue over there!

**Pull request closed without merging**